pageserver/src/import_datadir.rs

```diff
@@ -77,7 +77,7 @@ pub async fn import_timeline_from_postgres_datadir(
             let mut file = tokio::fs::File::open(absolute_path).await?;
             let len = metadata.len() as usize;
             if let Some(control_file) =
-                import_file(&mut modification, relative_path, &mut file, len, ctx).await?
+                import_file(&mut modification, relative_path, &mut file, len, false, ctx).await?
```

Comment:
Why not import aux files in this case?

Reply:
It was necessary to pass some test. Also comment says that this function is called only for directory initialized by initdb. There is no need to import any AUX files even if they are stored ion shutdown checkpoint.
```
 This is currently only used to import a cluster freshly created by initdb.
 ```